### PR TITLE
v5.10.0

### PR DESCRIPTION
**Pull request by y4ssi**

### CODE REVIEW

**Review by str4d — APPROVED** [review]

File: getaddressmempool.html [hunk 1]
```diff
@@ -36,21 +36,6 @@ <h1>getaddressmempool - Zcash 5.5.0 RPC</h1>
 
 Returns all mempool deltas for an address.
 
-WARNING: getaddressmempool is disabled.
```
Comment (str4d) [inline getaddressmempool.html:39]: Why did these disappear?

**Review by str4d — CHANGES_REQUESTED** [review]

**Review by str4d — CHANGES_REQUESTED** [review]
The titles of all the pages still say "Zcash 5.5.0 RPC Docs".

File: getaddressmempool.html (hunk 1 above)
Comment (y4ssi) [reply getaddressmempool.html:39]: I ran a test using `zcash-cli getaddressmempool` and at least it returns some value. I don't know if that warning was related to some version or if it's something that should be maintained

**Comment by y4ssi** [comment]
> The titles of all the pages still say "Zcash 5.5.0 RPC Docs".

Fixed

**Review by str4d — APPROVED** [review]

**Pull request merged**